types/types.d.ts

```diff
@@ -0,0 +1,202 @@
+import { TimeUnit, Format } from './constraints'
+
+export interface SpacetimeMain {
```

Comment:
there may be a way to import these types from spacetime somehow, or maybe there isn't. Ideally, they don't have to be maintained in both repos? Not a big deal to me!

Reply:
yea, sure, the only difference is adding one optional property on line `198`
```
export interface TimezoneSet {
  [key: string]: {
    offset: number
    hem: string
    dst?: string
    ctry?: string <<<
  }
```
I added it here so you know this file from `spacetime` was changed as well (together with `./zonefile/iana.js`.